proof/invariant-abstract/DetSchedSchedule_AI.thy

```diff
@@ -14391,7 +14349,8 @@ lemma send_signal_BOR_helper:
                       maybe_donate_sc_released_ipc_queues'
                 simp: valid_sched_def)
    apply (wpsimp cong: conj_cong simp: obj_at_kh_kheap_simps)
-  apply (clarsimp simp: valid_sched_def obj_at_kh_kheap_simps vs_all_heap_simps current_time_bounded_def)
+  apply (clarsimp simp: valid_sched_def obj_at_kh_kheap_simps vs_all_heap_simps current_time_bounded_def
+                  dest!: valid_ready_qs_etcb_eq)
```

Comment:
This indentation is fine, of course, but I thought you preferred lining up the colons